### ISSUE TITLE
Support for infinite scrollback lines

### ISSUE THREAD

**Issue by Dennis360ict** [asbru-cm/asbru-cm#739]
Hi, love asbru! thanks for the work. 

I would like to have an infinite scrolling option for the line buffers in an ssh session, like in terminator. asbru seems to have only the setting to set a max number of srollback lines under "edit connection" under "terminal options" which has the option to set the number of scrollback lines:
![image](https://user-images.githubusercontent.com/12065502/100456730-286d3880-30c1-11eb-9264-23ad5496368b.png)

I have it set to the max (default seems to be 5000?) of 99999, but I think it is still limited to 5000 lines. Even if i set it to 10000, the results ends the same. I tested with `cat -n` with a file that has 22000 lines of code. 

So my questions are these:
- can you confirm I can't set the number of lines in a session. Is this a bug or feature?
- can I request a feature with the option of infinite scrolling?

**Comment by gfrenoy** [comment]
Thanks for your positive feedback and your suggestion !

This value is passed to VTE API `set_scrollback_lines`, according to the documentation (see [here](https://developer.gnome.org/vte/unstable/VteTerminal.html#vte-terminal-set-scrollback-lines)), a negative value means `infinite scrollback`.

But the UI does not accept negative value for the moment ... I believe we should accept:
 - 0 : disable scrolling
 - -1 : infinite scrolling

**Comment by gfrenoy** [comment]
The last commit should fix the bug : Ásbrú only considered the global preference for scrollback lines.  The same option in terminal options was ignored.

I also added the possibility to select `0` or `-1` as the value for `scrollback lines`.

The build will be ready in about 30 minutes.

Can you give it a try using our snapshot build and let us know if it now works as expected (see https://docs.asbru-cm.net/General/Installation/) ?

Thanks again for your contribution.

**Comment by Dennis360ict** [comment]
Wow, this was a really fast reaction. I'm impressed! Thank you!

So I upgraded to the snapshot version and im running 6.2.3:
https://github.com/asbru-cm/asbru-cm/milestone/9

I can confirm:
- changing the scrollback lines from 5000 to 20000 in an connection works. I needed to reconnect before the settings became active (which is normal I think, just mentioning it here).
- changing to -1 in an connection works too (reconnect necessary of course).
- changing to -2 in an connection works too (reconnect necessary)
